src/hooks/useConcepts.tsx

```diff
@@ -5,10 +5,31 @@ const conceptRepresentation =
   'custom:(uuid,display,conceptMappings:(conceptReferenceTerm:(conceptSource:(name),code)))';
 
 export function useConcepts(references: Set<string>) {
-  // TODO: handle paging (ie when number of concepts greater than default limit per page)
-  const { data, error, isLoading } = useSWRImmutable<{ data: { results: Array<OpenmrsResource> } }, Error>(
+  const pageSize = 50;
+   // TODO: handle paging (ie when number of concepts greater than default limit per page)
+   const { data, error, isLoading } = useSWRImmutable<{ data: { results: Array<OpenmrsResource> } }, Error>(
     `/ws/rest/v1/concept?references=${Array.from(references).join(',')}&v=${conceptRepresentation}`,
     openmrsFetch,
   );
+
+  const fetchConcepts = async (url:string) => {
```

Comment:
Yes, agree with @samuelmale ... this function never seems to be called, @sibendire  have you tested that this actually works?

Reply:
It also looks like we are no longer using SWR... we would want to preserve the usage of SWR here....

Reply:
No worries @sibendire  but I don't see any new changes?

Reply:
Hello @mogoodrich  I believe with your guidance I can make as it expect. Could you review just slight change made and give me the feedback on how I have been missing up to fix the issue ?

Thank you.

Reply:
Am looking forward to @mogoodrich @samuelmale and @denniskigen for your response and guidance that will teach me more about this.
Thank you.